src/hvf/src/lib.rs

```diff
@@ -156,11 +206,56 @@ pub fn vcpu_set_vtimer_mask(vcpuid: u64, masked: bool) -> Result<(), Error> {
     }
 }
 
+pub struct HvfNestedBindings {
+    hv_vm_config_get_el2_supported:
+        libloading::Symbol<'static, unsafe extern "C" fn(*mut bool) -> hv_return_t>,
+    hv_vm_config_set_el2_enabled:
+        libloading::Symbol<'static, unsafe extern "C" fn(hv_vm_config_t, *mut bool) -> hv_return_t>,
+}
+
 pub struct HvfVm {}
 
+static HVF: LazyLock<libloading::Library> = LazyLock::new(|| unsafe {
+    libloading::Library::new(
+        "/System/Library/Frameworks/Hypervisor.framework/Versions/A/Hypervisor",
+    )
+    .unwrap()
```

Comment:
Does it make sense to fail gracefully here in the case that the `Hypervisor` library is either unavailable or of a different version?

Reply:
That'd be an ABI break on the macOS side, it's ~ the equivalent of .so versions on Linux

Reply:
Well, in the unlikely case Apple stops shipping version 'A' of Hypervisor.framework, we can't do much other than printing a different error message. ;-P